docker/de/Readme.md

```diff
@@ -17,7 +17,7 @@ $ sh bin/build-docker-de
 >**Note:** The `--name` flag allows you to name the container and refer to that name in subsequent commands.
 
 ```sh
-$ docker run --rm -p 1999:1999 --name chaincore chaincore/developer
+$ docker run -p 1999:1999 --name chaincore chaincore/developer
 ```
 
 By default, once you stop a running a container, all data is lost. Chain Core stores data in three locations: a data directory, a separate Postgres database, and a log directory. To persist the data, create directories on your development machine and mount them to the container on `docker run`:
```

Comment:
Technically, the phrase "By default, once you stop a running a container, all data is lost." is incorrect. The data is lost once you remove the container from the Docker host. Would you mind updating that in this PR.